packages/tracing/test/utils.test.ts

```diff
@@ -1,4 +1,21 @@
-import { extractTraceparentData } from '../src/utils';
+import { extractTraceparentData, hasTracingEnabled } from '../src/utils';
+
+describe('hasTracingEnabled', () => {
+  const tracesSampler = () => 1;
+  const tracesSampleRate = 1;
+  it.each([
+    ['no options', undefined, false],
```

Comment:
Doesn't the output in this one case depend on the return value of `getCurrentHub()`?

Reply:
Yes it depends on `getCurrentHub`, but there is no client attached to it during the test, so it works as expected.

I agree it could be a one liner, but I didn't change the function to minimize changing behaviour. I can go back and update!

Reply:
changed